system/lib/wasmfs/file.h

```diff
@@ -263,6 +222,12 @@ class Directory : public File {
       return {};
     }
   }
+
+protected:
+  std::vector<Entry> entries;
```

Comment:
Let's at least add a comment here saying that a vector saves code size but does lead to linear time in certain operations etc.

Reply:
Sounds good!